build/gn_run_binary.py

```diff
@@ -20,7 +20,7 @@
 args = [path] + sys.argv[2:]
 
 try:
-  subprocess.check_output(args, stderr=subprocess.STDOUT, universal_newlines=True)
```

Comment:
`universal_newlines` has a bunch of side-effects IIRC. Would you mind double-checking the docs to make sure we don't need them?